### ISSUE TITLE
password list contains control character

### ISSUE THREAD

**Issue by trivalik** [danielmiessler/SecLists#139]
The file    10_million_password_list_top_1000000.txt   contains ETX
on line 912955
this should be removed.

Edit:

3 passwords have this ETX.

**Comment by g0tmi1k** [comment]
```
$ cat -n 10-million-password-list-top-1000000.txt | grep 912955
912955	WORROM4
$
$ cat -v -n 10-million-password-list-top-1000000.txt | grep 912955
912955	WORROM^C4
$
$ cat -v -n 10-million-password-list-top-1000000.txt | grep '\^C4'
912955	WORROM^C4
916920	wolfpack^C4
$
```

**Comment by g0tmi1k** [comment]
Fixed in #180
